frontend/apps/desktop/src/editor/hovering-toolbar.tsx

```diff
@@ -50,8 +43,8 @@ import {MARK_STRONG} from './strong'
 import {MARK_UNDERLINE} from './underline'
 import {isMarkActive, toggleFormat} from './utils'
 
-export function EditorHoveringToolbar() {
-  const {editor} = useSlateWithV()
+export function EditorHoveringToolbar(mouseDown: boolean) {
+  const editor = useSlate()
```

Comment:
not really sure why this is in the Tamagui branch but looks good

Reply:
> I guess typescript is updated or is generally more strict? And that is why all the @tsignores are added? Or those ts errors were already there?

I guess TS is more strict now. that's why there's a bunch of tsignores now. we should avoid does when we refactor code bit by bit.

Reply:
> not really sure why this is in the Tamagui branch but looks good

this is there because I constantly merge master with it. and this is in master